src/directives/popup.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @fileoverview Provides a directive used to show a popup over the page with
+ * a title and content.
+ * The directive doesn't create any scope but relies on its parent scope.
+ * Properties like 'content', 'title' or 'open' come from the parent scope.
+ *
+ * This directive is intented to be used along with the popup service.
```

Comment:
You probably want to mention the templateUrl, like we do for other widget directives.